lumicks/pylake/kymotracker/tests/test_refinement.py

```diff
@@ -6,11 +6,13 @@
 
 
 def test_kymoline_interpolation():
-    time_idx = [1.0, 3.0, 5.0]
-    coordinate_idx = [1.0, 3.0, 3.0]
+    time_idx = np.array([1.0, 3.0, 5.0])
+    coordinate_idx = np.array([1.0, 3.0, 3.0])
     interpolated = KymoLine(time_idx, coordinate_idx, []).interpolate()
     assert np.allclose(interpolated.time_idx, [1.0, 2.0, 3.0, 4.0, 5.0])
     assert np.allclose(interpolated.coordinate_idx, [1.0, 2.0, 3.0, 3.0, 3.0])
+    assert isinstance(interpolated.time_idx, np.ndarray)
+    assert isinstance(interpolated.coordinate_idx, np.ndarray)
```

Comment:
You are correct. We don't need it to be an array-like.

I should test the behaviour that I was trying to saveguard with this, rather than the type. I've added a small extra test, to check whether lines can still be concatenated after refinement / interpolation.